lib/src/builder/parsing/declarations_from_members.dart

```diff
@@ -280,13 +289,46 @@ Iterable<BoilerplateDeclaration> getBoilerplateDeclarations(
         consume(factory);
         consumePropsAndState();
         yield PropsMapViewOrFunctionComponentDeclaration(
-          factory: factory,
+          factories: [factory],
           props: propsClassOrMixin,
         );
       }
     }
   }
 
+  if (_functionComponentFactories.isNotEmpty) {
+    final allUnusedProps = [
+      members.props,
+      members.propsMixins,
+    ].expand((i) => i).whereNot(hasBeenConsumed);
+
+    final standAloneFactories = _functionComponentFactories.where((factory) =>
+        _getPropsNameFromFunctionComponent(factory) == 'UiProps' || !_hasConfigArg(factory));
+    for (final factory in standAloneFactories) {
+      consume(factory);
+      yield PropsMapViewOrFunctionComponentDeclaration(
+        factories: [factory],
+        props: null,
```

Comment:
If we don't need to generate anything for these cases, can we instead not `yield` them? That's what we do for other declarations that don't need generation.

Also, for other declarations, we use a confidence of `neutral` so that the line
```dart
if (!resolveVersion(nonNullFactoryPropsOrComponents).shouldGenerate) continue;
```
below bails out of generation, as opposed to explicitly consuming them.

Can we update the logic around whether a factory is standalone into its initial confidence computation within `members_from_ast.dart`, so that it's consistent with how other members are handled?

Reply:
Oh that makes sense! I'll update that!